src/Controller/AccessManagementController.php

```diff
@@ -49,15 +58,19 @@ public function index()
      */
     public function userGroups()
     {
-        return $this->render('accessmanagement/groups.html.twig');
+        return $this->render($this->contextTemplate->GetProgramTemplate('accessmanagement/groups.html.twig'));
     }
 
     /**
      * @Route("/user/group/{groupId}", name="access_manage_user_group")
      */
     public function userGroup($groupId): Response
     {
-        $group = $this->getSecurityUser()->getGroupFromId($groupId);
+        if ($this->requestStack->getSession()->get('program') === 'hpo') {
```

Comment:
Instead of using magic strings "hpo" and "nph", let's reference a constant.  Since we're calling this same logic frequently, it would also be a good idea to create `isHealthPro` and `isNph` tests in the `BaseController` or as static methods in some other service.

Reply:
Will make a seperate issue to implement this.